core/lib/zksync_core/src/api_server/web3/mod.rs

```diff
@@ -300,7 +300,6 @@ impl<G: 'static + Send + Sync + L1GasPriceProvider> ApiBuilder<G> {
             sync_state: self.sync_state,
             api_config: self.config,
             last_sealed_miniblock,
-            logs_translator_enabled: self.logs_translator_enabled,
```

Comment:
You forgot to remove  `logs_translator_enabled`  field from `ApiBuilder<G>` struct

Reply:
Good catch, done!